ushahidi/platform-client
# White overlay in user search

This pull request makes the following changes:
- Removes the search-dropdown after search (the one that appears when typing in the filter-input-field)
- Fixes the clear-input-field button

Testing checklist:
- Go to 'Settings-Users'
- Click on search bar to enter name to search for
- Enter something
- Large white overlay appears from the search bar
- Click **enter** to perform search
- [ ] The white overlay disappears and the results appear in the userlist
- Click on search bar to enter name to search for
- Enter something
- Large white overlay appears from the search bar
- Click on the **button** in the dropdown to perform search
- [ ] The white overlay disappears and the results appear in the userlist
- Click on the **button** in the dropdown to perform search

- Click on search bar to enter name to search for
- Enter something
- Click on the x in the searchbar
- [ ] The input-field is emptied
- Click on search bar to enter name to search for
- Enter something
- Perform search through enter or click button
- Search-results appear
- Click on the x in the searchbar
- [ ] The input-field is emptied
- [ ] The userlist is back to its initial state



Fixes ushahidi/platform#3649 .

Ping @ushahidi/platform
